docs/internals/deprecation.txt

```diff
@@ -19,6 +19,8 @@ details on these changes.
 
 .. _deprecation-removed-in-5.0:
 
+* The Model Meta ``Options.index_together`` option will be removed.
```

Comment:
Misplaced, move above `.. _deprecation-removed-in-5.0:`

Reply:
```suggestion
* The model's ``Meta.index_together`` option will be removed.
```